Modules/_sqlite/connection.c

```diff
@@ -1080,19 +1127,26 @@ static PyObject* pysqlite_connection_set_trace_callback(pysqlite_Connection* sel
 }
 
 #ifndef SQLITE_OMIT_LOAD_EXTENSION
-static PyObject* pysqlite_enable_load_extension(pysqlite_Connection* self, PyObject* args)
+/*[clinic input]
+_sqlite3.Connection.enable_load_extension as pysqlite_connection_enable_load_extension
+
+    enable as onoff: int
```

Comment:
`bool(accept={int})`

Semantically it is a bool.

Reply:
> `bool(accept={int})`

Nice trick! Thanks

Fixed by @corona10 in #23837